lib/jekyll-gist/gist_tag.rb

```diff
@@ -42,6 +47,16 @@ def gist_script_tag(gist_id, filename = nil)
         end
       end
 
+      def gist_noscript_tag(gist_id, filename = nil)
+        if filename.empty?
+          uri = "https://gist.githubusercontent.com/#{gist_id}/raw"
+        else
+          uri = "https://gist.githubusercontent.com/#{gist_id}/raw/#{filename}"
+        end
+        code = open(uri).read.chomp
```

Comment:
Ok! This is a great improvement and I'm excited to have it. Let me know if you need anything from me to get this done. :smiley: